sdks/browser-sdk/src/Client.ts

```diff
@@ -112,34 +112,34 @@ export class Client extends ClientWorkerClass {
     return this.#installationIdBytes;
   }
 
-  async #createInboxSignatureText() {
+  async createInboxSignatureText() {
     return this.sendMessage("createInboxSignatureText", undefined);
   }
 
-  async #addAccountSignatureText(newAccountAddress: string) {
+  async addAccountSignatureText(newAccountAddress: string) {
     return this.sendMessage("addAccountSignatureText", {
```

Comment:
Since this is public now I'd also add the allow reassignment Boolean here as well we really don't want people accidentally rotating inbox ids.

Reply:
thinking about this, i'm not sure we need that type of guard here since it's just returning a string and not actually doing anything. WDYT?